instrumentation/instaredis/instaredis_test.go

```diff
@@ -0,0 +1,347 @@
+// (c) Copyright IBM Corp. 2022
+
+//go:build go1.13
+// +build go1.13
```

Comment:
Same as above, why is this limited?

Reply:
As explained previously, go-redis relies on resources available only in go1.13.